onboarding/transport/http_client.go

```diff
@@ -0,0 +1,37 @@
+package transport
+
+import (
+	"context"
+
+	platform "github.com/influxdata/influxdb/v2"
+	"github.com/influxdata/influxdb/v2/pkg/httpc"
+)
+
+// FIXME: Ideally we wouldn't need to keep this hand-rolled HTTP client in the influxdb codebase now that we've
```

Comment:
Is there already an issue created for this? Seems like there might be but I can't remember exactly. Would be good to create one if there isn't already one so we can clean this up.

Reply:
There is #21801